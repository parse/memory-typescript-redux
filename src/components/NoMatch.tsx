import * as React from 'react';

function NoMatch({  }: any) {
  return <p>Sorry, page not found</p>;
}

export default NoMatch;
