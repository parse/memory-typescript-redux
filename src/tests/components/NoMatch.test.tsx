import * as React from 'react';
import NoMatch from '../../components/NoMatch';
import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<NoMatch />).toJSON();
  expect(tree).toMatchSnapshot();
});
