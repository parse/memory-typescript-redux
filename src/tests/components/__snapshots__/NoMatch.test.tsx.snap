// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p>
  Sorry, page not found
</p>
`;
